src/pybind/mgr/dashboard/frontend/src/styles/ceph-custom/_basics.scss

```diff
@@ -97,3 +97,37 @@ option {
 .border-success {
   border-left: 4px solid vv.$success;
 }
+
+.vertical-line {
+  border-left: 1px solid vv.$gray-400;
+}
+
+.accordion {
+  .card {
+    border: 0;
+  }
+
```

Comment:
Looks like [Bootstrap 5 brings an accordion](https://getbootstrap.com/docs/5.0/components/accordion/) class (not BS 4 though). If not, is this somehow based on the Patternfly accordion style?

Reply:
I used the [BS4 accordion.](https://getbootstrap.com/docs/4.6/components/collapse/#accordion-example). Its not easy to find though. It was hidden way inside the collapse component.

Reply:
ok!